squirrels/_initializer.py

```diff
@@ -172,4 +172,6 @@ def get_content(file_name: Optional[str]) -> str:
                 self._copy_database_file(c.WEATHER_DB_NAME+".db")
             else:
                 raise NotImplementedError(f"No database found called '{sample_db}'")
+        elif sample_db is None:
+            self._copy_database_file(c.EXPENSES_DB_NAME+".db")
```

Comment:
Although correct, this requires maintaining "copy expenses db" logic in two different places